docs/guides/external-mgs/readme.md

```diff
@@ -0,0 +1,149 @@
+  lustreStorage:
+    externalMgs: ""
+    combinedMgtMdt: false
+    standaloneMgtPoolName: ""
+[...]
+```
+
+The persistent storage is created with the following DW directive:
+
+```bash
+#DW create_persistent name=shared-lustre capacity=100GiB type=lustre profile=persistent-lustre-shared-mgt
+```
+
+After the persistent Lustre file system is created, an admin can discover the MGS address by looking at the `NnfStorage` resource with the same name as the persistent storage that was created (`shared-lustre` in the above example).
+
+```yaml
+apiVersion: nnf.cray.hpe.com/v1alpha1
+kind: NnfStorage
+metadata:
+  name: shared-lustre
+  namespace: default
+[...]
+status:
+  mgsNode: 5.6.7.8@eth1
+[...]
+```
+
+A separate `NnfStorageProfile` can be created that specifies the MGS address.
+
+```yaml
+apiVersion: nnf.cray.hpe.com/v1alpha1
+kind: NnfStorageProfile
+metadata:
+  name: internal-mgt
+  namespace: nnf-system
+data:
+[...]
+  lustreStorage:
+    externalMgs: 5.6.7.8@eth1
+    combinedMgtMdt: false
+    standaloneMgtPoolName: ""
+[...]
+```
+
+With this configuration, an admin must determine that no file systems are using the shared MGT before destroying the persistent Lustre instance.
+
+## Configuration with an Internal MGT Pool
+
+Another method NNF supports is to create a number of persistent Lustre MGTs on NNF nodes. These MGTs are not part of a full file system, but are instead added to a pool of MGTs available for other Lustre file systems to use. Lustre file systems that are created will choose one of the MGTs at random to use and add a reference to make sure it isn't destroyed. This configuration has the advantage of spreading the Lustre management load across multiple servers. The disadvantage of this configuration is that it does not provide high availability.
+
+To configure the system this way, the first step is to make a pool of Lustre MGTs. This is done by creating a persistent instance from a storage profile that specifies the `standaloneMgtPoolName` option. This option tells NNF software to only create an MGT, and to add it to a named pool. The following `NnfStorageProfile` provides an example where the MGT is added to the `example-pool` pool:
+
+```yaml
+apiVersion: nnf.cray.hpe.com/v1alpha1
+kind: NnfStorageProfile
+metadata:
+  name: mgt-pool-member
+  namespace: nnf-system
+data:
+[...]
+  lustreStorage:
+    externalMgs: ""
+    combinedMgtMdt: false
+    standaloneMgtPoolName: "example-pool"
+[...]
+```
+
+A persistent storage MGTs can be created with the following DW directive:
+
+```bash
+#DW create_persistent name=mgt-pool-member-1 capacity=1GiB type=lustre profile=mgt-pool-member
+```
+
+Multiple persistent instances with different names can be created using the `mgt-pool-member` profile to add more than one MGT to the pool. To create a Lustre file system that uses one of the MGTs from the pool, an `NnfStorageProfile` should be created that uses the special notation `pool:[pool-name]` in the `externalMgs` field.
```

Comment:
I think I'd start new paragraph after the first sentence since you're illustrating on two different points. Just personal preference, though.

```suggestion
Multiple persistent instances with different names can be created using the `mgt-pool-member` profile to add more than one MGT to the pool.

To create a Lustre file system that uses one of the MGTs from the pool, an `NnfStorageProfile` should be created that uses the special notation `pool:[pool-name]` in the `externalMgs` field.
```